charts/alfresco-adf-app/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.configMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "alfresco-adf-app.fullname" . }}
+  labels:
+    {{- include "alfresco-adf-app.labels" . | nindent 4 }}
+data:
+  config.json: >-
```

Comment:
not sure if we can hardcode that and if we need to add the volume mounts but I think its better if we leave volume mounts optional

Reply:
not only can we but we have to. However it should be named `app.config.json` IIRC